src/transactions/feed/JumpstartTransactionDetailsScreen.tsx

```diff
@@ -81,6 +89,7 @@ async function getClaimDataAndStatus(
 function JumpstartTransactionDetailsScreen({ route }: Props) {
   const { transaction } = route.params
   const { t } = useTranslation()
+  const dispatch = useDispatch()
 
   const [reclaimTx, setReclaimTx] = useState<SerializableTransactionRequest | null>(null)
```

Comment:
can we remove this state variable and use fetchClaimData.result?.preparedTransaction instead?